prov/shm/src/smr_progress.c

```diff
@@ -200,6 +207,75 @@ static int smr_progress_iov(struct smr_cmd *cmd, struct iovec *iov,
 	return -ret;
 }
 
+static int smr_mmap_peer_copy(struct smr_ep *ep, struct smr_cmd *cmd,
+				 struct iovec *iov, size_t iov_count,
+				 size_t *total_len)
+{
+	char shm_name[NAME_MAX];
+	void *mapped_ptr;
+	int peer_id, fd, ret, num;
+
+	peer_id = (int) cmd->msg.hdr.addr;
+
+	num = snprintf(shm_name, NAME_MAX - 1, "%s_%s_%ld",
+		       ep->region->map->peers[peer_id].peer.name, ep->name,
+		       cmd->msg.hdr.msg_id);
```

Comment:
I would make this a common inline so that if we change the format for the mmap file name, we only have it in one place

Reply:
Done.